bundle/src/main/java/com/adobe/acs/commons/granite/ui/components/include/IncludeDecoratorFilterImpl.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * #%L
+ * ACS AEM Commons Bundle
+ * %%
+ * Copyright (C) 2017 Adobe
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.adobe.acs.commons.granite.ui.components.include;
+
+import org.apache.commons.collections.MapUtils;
+import org.apache.sling.api.SlingHttpServletRequest;
+import org.apache.sling.api.resource.Resource;
+import org.apache.sling.api.resource.ValueMap;
+import org.apache.sling.engine.EngineConstants;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.ConfigurationPolicy;
+
+import javax.annotation.CheckForNull;
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+
+
+import java.io.IOException;
+
+@Component(
+        service = Filter.class,
+        configurationPolicy = ConfigurationPolicy.OPTIONAL,
+        property= {
+                EngineConstants.SLING_FILTER_SCOPE+"="+ EngineConstants.FILTER_SCOPE_INCLUDE,
+                "sling.filter.resourceTypes=" + IncludeDecoratorFilterImpl.RESOURCE_TYPE
+        }
+        
+)
+public class IncludeDecoratorFilterImpl implements Filter {
+
+    static final String RESOURCE_TYPE = "acs-commons/granite/ui/components/include";
+    static final String NAMESPACE = "namespace";
+    static final String PARAMETERS = "parameters";
+    public static final String PREFIX = "PREFIX_";
```

Comment:
Would it hurt to make this value very "ACS Commons" specific just to "guarantee" no collisions?
... like `ACS_AEM_COMMONS__NAMESPACE_INCLUDE_` or something?